roles/test-utils/mining-device/README.md

```diff
@@ -3,16 +3,28 @@
 Header only sv2 cpu miner.
 
 ```
-Usage: mining-device [OPTIONS] --address-pool <ADDRESS_POOL>
+Usage: mining_device [OPTIONS] --address-pool <ADDRESS_POOL>
 
 Options:
-  -p, --pubkey-pool <PUBKEY_POOL>    Pool pub key, when left empty the pool certificate is not checked
-  -i, --id-device <ID_DEVICE>        Sometimes used by the pool to identify the device
-  -a, --address-pool <ADDRESS_POOL>  Address of the pool in this format ip:port or domain:port
-      --handicap <HANDICAP>          This value is used to slow down the cpu miner, it represents the number of micro-seconds that are awaited between hashes [default: 0]
-      --id-user <ID_USER>            User id, used when a new channel is opened, it can be used by the pool to identify the miner
-  -h, --help                         Print help
-  -V, --version                      Print version
+  -p, --pubkey-pool <PUBKEY_POOL>
+          Pool pub key, when left empty the pool certificate is not checked
+  -i, --id-device <ID_DEVICE>
+          Sometimes used by the pool to identify the device
+  -a, --address-pool <ADDRESS_POOL>
+          Address of the pool in this format ip:port or domain:port
+      --handicap <HANDICAP>
+          This value is used to slow down the cpu miner, it represents the number of micro-seconds that are awaited between hashes [default: 0]
+      --id-user <ID_USER>
+          User id, used when a new channel is opened, it can be used by the pool to identify the miner
+      --nominal-hashrate-multiplier <NOMINAL_HASHRATE_MULTIPLIER>
```

Comment:
Adding a explanation of why this was added, when to use which %, and give an example of a 1,000 H/s miner use case.

Reply:
done